aries-site/src/examples/components/checkbox/CheckBoxValidationExample.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Box, CheckBox, Form, FormField } from 'grommet';
+
+export const CheckBoxValidationExample = () => {
+  const [togglemessage, setToggleMessage] = useState(
+    'Check toggle to resolve error',
+  );
+  const [message, setMessage] = useState('Check checkbox to resolve error');
+
+  return (
+    <Box gap="large" direction="row">
+      <Box width="medium" gap="medium">
+        <Form>
+          <FormField
+            name="required-field"
+            label="Label"
+            htmlFor="required-field"
+            error={message}
```

Comment:
Seems like if I uncheck the checkbox again that the error should re-appear.

Reply:
yeah I was thinking the same Im not sure why that state is not being set again. I can look further this would be the direction we want thought before I do so ? We want to be able to go back and forth with error and no error